source/Engine.h

```diff
@@ -251,6 +251,9 @@ class Engine {
 	double load = 0.;
 	int loadCount = 0;
 	double loadSum = 0.;
+
+	// For keeping track of which ships were repaired in their carrier's bays
+	std::vector<std::pair<std::shared_ptr<Ship>,std::shared_ptr<Ship>>> repairedInBay;
```

Comment:
This should be moved, I think it would fit to the newShips and that stuff

Reply:
I moved it. How does it look now?